packages/plugin-exec/sources/ExecFetcher.ts

```diff
@@ -94,6 +107,41 @@ export class ExecFetcher implements Fetcher {
         const stdout = xfs.createWriteStream(logFile);
         const stderr = stdout;
 
+        /**
+         * Values exposed on the global `execEnv` variable.
+         *
+         * Must be stringifiable using `JSON.stringify`.
+         */
+        const execEnvValues: Partial<ExecEnv> = {
+          tempDir: npath.fromPortablePath(cwd),
+          locator,
```

Comment:
Rather than a raw locator, I think we should just pass the string representation (`structUtils.stringifyLocator`)